src/Components/profileOutline.js

```diff
@@ -0,0 +1,208 @@
+    <div style={styles.text}>
+        {pronouns} 
+    </div>
+        <br>
+        </br>
+        <div style={styles.header}>
+        Email:
+    </div>
+    <div style={styles.text}>
+        {email} 
+    </div>
+        <br>
+        </br>
+        <div style={styles.header}>
+        Phone:
+    </div>
+    <div style={styles.text}>
+        {phone}
+    </div> 
+        <br>
+        </br>
+    <div style={styles.header}>
+        City:
+    </div>
+    <div style={styles.text}>
+        {city} 
+    </div>
+        <br>
+        </br>
+    <div style={styles.header}>
+        State:
+    </div>
+    <div style={styles.text}>
+        {state} 
+    </div>
+        <br>
+        </br>
+        <div style={styles.header}>
+        Ethnicity:
+    </div>
+    <div style={styles.text}>
+        {ethnicity} 
+    </div>
+        <br>
+        </br>
+    <div style={styles.header}>
+        Age:
+    </div>
+    <div style={styles.text}>
+        {age}
+    </div> 
+        <br>
+        </br>
+    <div style={styles.header}>
+        Gender Identity:
+    </div>
+    <div style={styles.text}>
+        {genderIdentity} 
+    </div>
+        <br>
+        </br>
+    <div style={styles.header}>
+        Sexuality:
+    </div>
+    <div style={styles.text}>
+        {sexuality} 
+    </div>
+        <br>
+        </br>
+        <div style={styles.header}>
+        Veteran:
+    </div>
+    <div style={styles.text}>
+        {veteran} 
+    </div>
+        <br>
+        </br>
+        <div style={styles.header}>
+        Disability:
+    </div>
+    <div style={styles.text}>
+        {disability}
+    </div> 
+        <br>
+        </br>
+    <div style={styles.header}>
+        Housing Situation:
+    </div>
+    <div style={styles.text}>
+        {housingSituation} 
+    </div>
+        <br>
+        </br>
+    <div style={styles.header}>
+        Employment Status:
+    </div>
+    <div style={styles.text}>
+        {employmentStatus} 
+    </div>
+        <br>
+        </br>
+        <div style={styles.header}>
+        Prior Convictions:
+    </div>
+    <div style={styles.text}>
+        {priorConvictions} 
+    </div>
+        <br>
+        </br>
+</div>
+
+)
+
+
+
+// end of function bracket 
+}
+
+ProfileOutline.propTypes = {
+    name: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    pronouns: PropTypes.string.isRequired,
+    email: PropTypes.string.isRequired,
+    phone: PropTypes.string.isRequired,
+    city: PropTypes.string.isRequired,
+    state: PropTypes.string.isRequired,
+    ethnicity: PropTypes.string.isRequired,
+    age: PropTypes.number.isRequired,
+    genderIdentity: PropTypes.string.isRequired,
+    sexuality: PropTypes.string.isRequired,
+    veteran: PropTypes.string.isRequired,
+    disability: PropTypes.string.isRequired,
+    housingSituation: PropTypes.string.isRequired,
+    employmentStatus: PropTypes.string.isRequired,
+    priorConvictions: PropTypes.string.isRequired,
+  }
```

Comment:
FANTASTIC JOB ON YOUR FIRST SPRINT!!! GO ALAN AND KAYLEE!! You guys worked really well together, and sprinted super efficiently! The logic looks good, I think now we can just clean it up a bit. 

I think the 2 main things are that the props can be passed as an object, to cut down on lines of code! The other thing is that I think we can pull out the demographic info as a separate component that can be used by the profile page, just to increase modularity!

Other than that it looks really good and I am soso proud!